ui/apps/platform/src/Containers/Integrations/utils/integrationsList.ts

```diff
@@ -84,6 +84,13 @@ export const imageIntegrationsDescriptors: ImageIntegrationDescriptor[] = [
         label: 'StackRox Scanner',
         type: 'clairify',
     },
+    {
+        categories: 'Scanner',
+        image: logo,
+        label: 'StackRox Scanner V4',
```

Comment:
minor nit:


Consider changing the tile name from `StackRox Scanner V4` to just `Scanner V4` if appropriate to do so

On the System Health page the new scanner has name `Scanner V4`:

![image](https://github.com/stackrox/stackrox/assets/119438707/97e5db08-497f-4c9c-baaf-6a78c0d82ba1)

![image](https://github.com/stackrox/stackrox/assets/119438707/825efec2-6cd6-41c4-8468-86924d20c48d)